lms/static/sass/shared-v2/_footer.scss

```diff
@@ -2,199 +2,198 @@
 // ====================
 
 .wrapper-footer {
-  @extend %ui-print-excluded;
-  margin-top: ($baseline*2);
-  box-shadow: 0 -1px 5px 0 $shadow-l1;
-  border-top: 1px solid tint(palette(grayscale, light), 50%);
-  padding: 25px ($baseline/2) ($baseline*1.5) ($baseline/2);
-  background: $footer-bg;
-  clear: both;
-
-  footer#footer-openedx {
-    @include clearfix();
-    box-sizing: border-box;
-    margin: 0 auto;
-
-    p, ol, ul {
-      font-family: $sans-serif;
-
-      // override needed for poorly scoped font-family styling on p a:link {}
-      a {
-        font-family: $sans-serif;
-      }
-    }
-
-    a {
-      @extend %link-text;
-      border-bottom: none;
-
-      &:hover,
-      &:focus,
-      &:active {
-        border-bottom: 1px dotted $link-color;
-      }
-    }
-
-    // colophon
-    .colophon {
-      @include span(12);
-
-      @include susy-media($bp-screen-sm) {
-        @include span(8);
-      }
-
-      .nav-colophon {
+    @extend %ui-print-excluded;
+    box-shadow: 0 -1px 5px 0 $shadow-l1;
+    border-top: 1px solid tint(palette(grayscale, light), 50%);
+    padding: 25px ($baseline/2) ($baseline*1.5) ($baseline/2);
+    background: $footer-bg;
+    clear: both;
```

Comment:
Just to be clear, I just reindented this file and didn't introduce this. I'm happy to make the changes though since you've done such a thorough review.